src/crustls.h

```diff
@@ -386,14 +386,18 @@ size_t rustls_version(char *buf, size_t len);
  */
 uint16_t rustls_supported_ciphersuite_get_suite(const struct rustls_supported_ciphersuite *supported_ciphersuite);
 
+/**
+ * Return the length of rustls' list of supported cipher suites.
+ */
+uintptr_t rustls_all_ciphersuites_len(void);
```

Comment:
Is `uintptr_t` what `cbindgen` generates for `usize`? Interesting. I think I might have used `size_t` or `off_t` if I were writing this C function by hand, but `uintptr_t` is close enough that it's not worth fighting with `cbindgen` over it.

Reply:
That's interesting. In most other cases, `cbindgen` generates `size_t`. There are two cases I see where it doesn't:

```
void rustls_client_session_get_alpn_protocol(const struct rustls_client_session *session,
                                             const uint8_t **protocol_out,
                                             uintptr_t *protocol_out_len);
```

(perhaps because it's a pointer?)

    uintptr_t rustls_all_ciphersuites_len(void);

(perhaps because it's a return value?)